aws/alarms/athena.tf

```diff
@@ -18,7 +18,7 @@ module "athena" {
 # Hold the Athena data
 #
 module "athena_bucket" {
-  source            = "github.com/cds-snc/terraform-modules?ref=v2.0.5//S3"
+  source            = "github.com/cds-snc/terraform-modules//S3?ref=v2.0.5"
```

Comment:
Any reason why we're still referencing a super old tag (2.0.5)?  Wondering if we can upgrade to latest tag version?

Reply:
It should be fine to upgrade, although some modules do have breaking changes.

I'll bump everything to latest and see what comes out in the plan.